app/domains/ranks/insanity_scores.rb

```diff
@@ -12,7 +12,7 @@ def ranking_criterion(score)
       completions = @completions.fetch(score[:user_id], [])
       [
         score[:score],
-        completions.count,
+        score[:score] == 0 ? score[:user_id] : completions.count,
         -completions.sum { |completion| completion.duration || 0 },
         -score[:user_id]
```

Comment:
At score 0, `completions.count` is also 0, and `completion.sum(&:duration)` also. Would've been easier to change it at line 17 by simply removing the `-` 😄

Reply:
En fait, je voulais que :
* quand le score est 0, le plus gros user_id soit mieux ranké
* quand le score est différent de 0, le plus petit user_id soit mieux ranké

Au final, je pense que la troisième condition est tellement rare (que 2 personnes aient le même temps total de résolution à la seconde près 🙄), qu'effectivement je suis safe à utiliser le dernier tie breaker pour le score à 0.

Je change dans une nouvelle PR, désolé pour le mic mac.

Reply:
Ahah, c'est pas suffisamment majeur pour nécessiter forcément d'intervenir. On nettoiera (ou pas) à l'occase, ça change pas grand chose.